src/Concerns/Schemata.php

```diff
@@ -300,7 +300,7 @@ public static function getSectionsSchema(): array
                     'default' => 1,
                     'md' => 2,
                     'xl' => 3,
-                    '2xl' => 4,
+                    '2xl' => 3,
```

Comment:
why change this? I think with `2x1`  there is enough space for 4 column

Reply:
I was trying to set 3 columns for 2xl and I forgot to set it back to 4.

Reply:
I will fix it NP